src/PostgREST/Config.hs

```diff
@@ -223,6 +227,7 @@ readOptions = do
           |
           |## base url for swagger output
           |# server-proxy-uri = ""
+          |# server-unix-socket = ""
```

Comment:
Could be confusing to offer the `""` value(since it's valid as I mentioned above) as an example. Perhaps the example should be: ` server-unix-socket = pgrst.sock`.

Also, when `server-port` and `server-unix-socket` are specified we prefer the unix socket. Perhaps you could add a comment to the config like: `If specified, server-unix-socket has precedence over server-port`.

Reply:
Thanks for the feedback. Both good points. I will update the PR asap.